static/app/utils/useProjectKeys.tsx

```diff
@@ -1,49 +1,28 @@
-import {useEffect, useState} from 'react';
-import * as Sentry from '@sentry/react';
+import type {ProjectKey} from 'sentry/types/project';
+import {
+  type ApiQueryKey,
+  useApiQuery,
+  type UseApiQueryOptions,
+} from 'sentry/utils/queryClient';
 
-import type {Organization, Project, ProjectKey, RequestState} from 'sentry/types';
+interface ProjectKeysParameters {
+  orgSlug: string;
```

Comment:
any reason why these are `string`s and the ones in other files are `Organization['slug']` and `Project['slug']`?

Reply:
No, both are strings. However, I prefer using `Organization['slug']` over `string`. This way, if the type changes, it will be reflected here. I missed this one earlier.

Reply:
yeah sounds good! just noticed the inconsistency. thanks for changing